views/scss/inc/_test-takers.scss

```diff
@@ -0,0 +1,92 @@
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU General Public License
+/// as published by the Free Software Foundation; under version 2
+/// of the License (non-upgradable).
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/// GNU General Public License for more details.
+///
+/// You should have received a copy of the GNU General Public License
+/// along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+/// Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+///
+
+@import "inc/bootstrap";
+
+
+.ttlist {
+    overflow: auto;
+}
+
+
+.ttlist li {
+    cursor: pointer;
+    border-bottom: 1px solid #ddd;
+    padding: 3px;
+    background:#F9F8F7;
```

Comment:
Where does this color come from ? Shouldn't we use only colors (and variations) from https://hub.taocloud.org/styleguide/colors ?